README.md

```diff
@@ -65,6 +67,8 @@ An ack/ag/pt powered code search and view tool, like ack.vim or `:vimgrep` but t
 
 6. `:CtrlSFOpen` can reopen CtrlSF window when you have closed CtrlSF window. It is free because it won't invoke a same but new search. A handy command `:CtrlSFToggle` is also available.
 
+7. Alternatively run `:CtrlSFLocation [pattern]`, it will only open a quickfix window to show search result.
```

Comment:
If only quickfix window is used here, maybe `CtrlSFQuickfix` is more intuitive. (hard to input though, please let me know if you have any better name!)